dev/com.ibm.ws.security.openidconnect.clients.common/resources/com/ibm/ws/security/openidconnect/clients/common/resources/OidcClientMessages.nlsprops

```diff
@@ -270,4 +270,27 @@ LOGOUT_TOKEN_MISSING_CLAIMS=CWWKS1545E: The logout token is not valid because it
 LOGOUT_TOKEN_MISSING_CLAIMS.explanation=The logout token must contain the claims that are specified in the message.
 LOGOUT_TOKEN_MISSING_CLAIMS.useraction=Ensure that the OpenID Connect provider includes the missing claims in the logout token.
 
+# Do not translate "sub" or "sid"
+LOGOUT_TOKEN_MISSING_SUB_AND_SID=CWWKS1546E: The logout token is not valid because it does not contain a "sub" or "sid" claim. The token must have at least one of those claims.
+LOGOUT_TOKEN_MISSING_SUB_AND_SID.explanation=The sub and sid claims identify the user and session that is associated with a token. At least one of those claims must be present to determine the user or session to log out.
```

Comment:
I'm not sure you need quotes around the claims in these messages, but if they're necessary, we should use the consistently 
```suggestion
LOGOUT_TOKEN_MISSING_SUB_AND_SID.explanation=The "sub" and "sid" claims identify the user and session that is associated with a token. At least one of those claims must be present to determine the user or session to log out.
```

Reply:
and so forth for the other messages in this set